Calling out support for Single-Region, Multi-Region, Multi-AZ for EKS (#26015)
cilium-team/cilium: Documentation/installation/requirements-eks.rst

.. tip::

>    - To chain Cilium on top of the AWS CNI, see :ref:`chaining_aws_cni`.
> 
>    - You can also bring up Cilium in a Single-Region, Multi-Region, or Multi-AZ environment for EKS.

Reviewer: No hyphens needed.
```suggestion
   To chain Cilium on top of the AWS CNI, see :ref:`chaining_aws_cni`.

   You can also bring up Cilium in a Single-Region, Multi-Region, or Multi-AZ environment for EKS.
```

>    - If you want to chain Cilium on top of the AWS CNI, refer to the guide :ref:`chaining_aws_cni`.
> 
>    - Cilium can also be brought up in a Single-Region, Multi-Region, Multi-AZ environment for EKS.

Reviewer: Edits for clarity, use active voice, address the user as "you"
```suggestion
   - To chain Cilium on top of the AWS CNI, see :ref:`chaining_aws_cni`.

   - You can also bring up Cilium in a Single-Region, Multi-Region, or Multi-AZ environment for EKS.
```
